### PR TITLE
Option-to-toggle-off-motion-on-mouseover_v2

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -8099,6 +8101,13 @@
   button.style.backgroundColor = framesPerUpdate == 60 ? "lightblue" : "";
 }
 
+function ClickStill() {
+  STATE_STILL = !STATE_STILL;
+  IN_FRAME = !IN_FRAME;
```
Comment (profezzorn) [inline style_editor.html:8106]: This seems really weird.
a) how can we be inside the frame if we just clicked this button?
b) why are we suddenly inside the frame afterwards?

Comment (NoSloppy) [reply style_editor.html:8106]: Admittedly, this is just a quick dirty way to just have the motion and background swing sin stop.
It does not have anything to do with actually being in the frame, it's just to take on the behavior as if we *were* inside the frame.

Comment (NoSloppy) [reply style_editor.html]: Ok I did it the right way instead.

**Comment by profezzorn** [comment]
I'm not really sure why we need this though, I mean if you don't want the saber to move, then just don't mouse over the image, right?

**Comment by NoSloppy** [comment]
While avoiding the canvas is of course a way to prevent the swinging animation, it seems to ALWAYS get rolled over, at least the corner, when going for menu items, or the top of the structured view etc...
Plus, this allows for any SwingSpeed layers to stop showing without needing to comment out the layer and re-submit.
A "danger" which is now less risky since comments stay, but in the past, commenting something to not see it for a moment inevitably meant losing it because one the right side is clicked, it' was gone.

**Comment by NoSloppy** [comment]
https://user-images.githubusercontent.com/53964195/228121070-2678cb09-91d9-4694-89f4-d3ee4bf921e2.mov

**Comment by profezzorn** [comment]
Wouldn't the swingspeed layers show anyways, even if the saber isn't moving?

**Comment by NoSloppy** [comment]
As in "shouldn't" they? I suppose so. 
The initial filed issue had separate elements, the swinging animation, and the background swingSpeed constantly running on a Sin.
This lumps them both onto one button to be less invasive with too many buttons. 
I don't know. Personally, I find it useful. I have had others test with positive feedback for the option to toggle the motion on rollover. 
If having SwingSpeed affected as well is illogical, I get that. Maybe we just stop the swing animation with the "Still" button, and leave the SwingSpeed up to just commenting out the layer?
I just don't know if users realize they can do that.

**Comment by NoSloppy** [comment]
Removed swingspeed from the Still button. 
Now it's just a motion toggle.

**Comment by NoSloppy** [comment]
Closing this in lieu of full revamp

**Pull request closed without merging**